Client/Frontend/Browser/BrowserViewController/ProductNotifications/BrowserViewController+ProductNotification.swift

```diff
@@ -27,23 +27,6 @@ extension BrowserViewController {
         case grandTier = 500_000
         case legendaryTier = 1_000_000
 
-        var title: String {
-            switch self {
-                case .specialTier:
-                    return Strings.ShieldEducation.benchmarkSpecialTierTitle
```

Comment:
I decided to leave those strings for now in the app, perhaps remove later or we might go back to them?